src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -891,7 +890,7 @@ const WorkflowView = _.flow(
                 ]),
                 div([
                   h(RadioButton, {
-                    disabled: !!WorkspaceUtils.editWorkspaceError(ws) || currentSnapRedacted,
+                    disabled: WorkspaceUtils.canEditWorkspace(ws).value || currentSnapRedacted,
```

Comment:
Still a few cases here with the logic flipped.

```suggestion
                    disabled: !WorkspaceUtils.canEditWorkspace(ws).value || currentSnapRedacted,
```